src/clr-angular/forms/datepicker/date-container.spec.ts

```diff
@@ -203,6 +218,12 @@ export default function() {
         controlClassService.className = 'clr-col-2';
         expect(context.clarityDirective.controlClass()).not.toContain('clr-col-md-10');
       });
+
+      it('clearDateValue should call method from DateIOService to request clear', () => {
```

Comment:
`DateFormControlService` not `DateIOService`

Reply:
Typo!